org.mozilla.Firefox.BaseApp.json

```diff
@@ -3,8 +3,35 @@
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "19.08",
     "sdk": "org.freedesktop.Sdk",
-    "separate-locales": false,
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/share/gtk-doc",
+        "/man",
+        "/share/man",
+        "*.la", "*.a"
```

Comment:
Also half of these are copy-pasted nonsense.

Reply:
Possibly, I wanted to be a future proof. Most of those were in [initial commit](https://github.com/flathub/org.mozilla.Firefox.BaseApp/commit/17403b19a79e7e95c3801d39e3421c984c8165b7#diff-2da58ed2a3ebc224d82f5248c8905b72R7).